Cargo.toml

```diff
@@ -10,22 +10,23 @@ version = "0.3.0"
 [features]
 default = []
 fluent = ["fluent-templates"]
+env_logger = []
 
 [dependencies]
 clap = { version = "4", features = ["derive", "env", "unicode", "cargo"] }
 color-eyre = "0.6"
-env_logger = "0.10"
-fluent-templates = { version = "0.8", optional = true, default-features = false, features = ["tera"]}
+env_logger = "0.11.3"
```

Comment:
Why switch to a patch-only version here? Pinning the minor version is generally a good idea so future `cargo update`s can update the patch version when available unless there is a known reason to avoid patch version updates (most Rust projects follow semver well enough for this not to be an issue).

Reply:
Sorry, my mistake. I removed and added env_logger while trying to understand the `nix build` error.